rcl/include/rcl/client.h

```diff
@@ -44,9 +46,15 @@ typedef struct rcl_client_options_s
 {
   /// Middleware quality of service settings for the client.
   rmw_qos_profile_t qos;
+  /// Publisher options for the service event publisher
+  rcl_publisher_options_t event_publisher_options;
   /// Custom allocator for the client, used for incidental allocations.
   /** For default behavior (malloc/free), use: rcl_get_default_allocator() */
   rcl_allocator_t allocator;
+  /// Enable/Disable service introspection features
+  bool enable_service_introspection;
+  /// The clock to use for service introspection message timestampes
```

Comment:
```suggestion
  /// The clock to use for service introspection message timestamps
```

Reply:
It might also be good to document if the clock is optional and what the behavior is if it is NULL.

Reply:
Fixed in 2f1e9fc252c60380f4157bbb01266f3ce87c2863.